<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_dialog</class>
 <widget class="QDialog" name="settings_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReconstructMeQt - Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/** @file
  * @copyright Copyright (c) 2012 PROFACTOR GmbH. All rights reserved. 
  *
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted provided that the following conditions are
  * met:
  *
  *     * Redistributions of source code must retain the above copyright
  * notice, this list of conditions and the following disclaimer.
  *     * Redistributions in binary form must reproduce the above
  * copyright notice, this list of conditions and the following disclaimer
  * in the documentation and/or other materials provided with the
  * distribution.
  *     * Neither the name of Google Inc. nor the names of its
  * contributors may be used to endorse or promote products derived from
  * this software without specific prior written permission.
  *
  * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
  * &quot;AS IS&quot; AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
  * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
  * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
  * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
  * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
  * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
  * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
  * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  * @authors christoph.heindl@profactor.at
  *                  christoph.kopf@profactor.at 
  *                  florian.eckerstorfer@profactor.at
  */
  

QWidget, QAbstractItemView::item, QMenuBar::item, QComboBox::drop-down
{
	border: 0px;
	font: 8pt;
    border-radius: 2px;
    padding: 4px;
    margin: 2px;
    color: rgb(180, 180, 180);
    background-color:  rgb(60, 60, 60);
}

QPushButton
{
    border: 1px solid  rgb(180,180, 180);
    min-width: 5em;
}

QPushButton:hover, QAbstractItemView::item:hover, QAbstractItemView::item:selected,  QCheckBox::indicator:checked, QMenu:selected, QMenu:hover,  QMenuBar::item:selected
{
    background-color:  rgb(80, 80, 80);
}

QLineEdit, QAbstractItemView, QCheckBox::indicator
{
   background-color:  rgb(50, 50, 50);
}

QCheckBox::indicator:checked
{
    image:url(:/images/checkbox.png);
}

/** -------------------------------
    * Scrollbar
    * ------------------------------ */


 QScrollBar:horizontal {
     margin: 0px 20px 0px 20px;
 }

 QScrollBar:vertical {
     margin: 20px 0px 20px 0px;
 }

 QScrollBar::handle,  QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal, QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
	 background-color:  rgb(180,180, 180);
     border: 1px solid  rgb(180,180, 180);
 } 

 QScrollBar::handle:horizontal {
     min-width: 10px;
 }

QScrollBar::handle:vertical {
     min-height: 10px;
 }

 QScrollBar::add-line:horizontal {
     width: 15px;
     height: 10px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:horizontal {
     width: 15px;
     height: 10px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }

 QScrollBar::add-line:vertical {
     height: 15px;
     width: 10px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     height: 15px;
     width: 10px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal,  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Config Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="config_path_tb"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse_button_config">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Sensor Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sensor_path_tb"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse_button_sensor">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>License File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="license_file_tb"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse_button_license">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Computation Device</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="device_list_widget">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ocl_device_auto_cb">
     <property name="toolTip">
      <string>Use this, if you want to ReconstructMe determine a fitting calcuation devcie</string>
     </property>
     <property name="text">
      <string>Auto select device</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ocl_device_auto_cb</sender>
   <signal>toggled(bool)</signal>
   <receiver>device_list_widget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ocl_device_auto_cb</sender>
   <signal>clicked()</signal>
   <receiver>device_list_widget</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
